src/components/ResetMapButton.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="leaflet-control-zoom leaflet-bar">
+        <a
+            @click.prevent="resetView"
+            class="leaflet-control-zoom-in"
+            href="#"
+            title="Reset map"
+            aria-label="Reset map"
+            role="button"
+        >
+            <vue-feather type="home" size="20" style="margin-top: 4px; margin-right: 1px"></vue-feather>
+        </a>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { useRouter } from "vue-router";
+import VueFeather from "vue-feather";
+import { defineProps } from "vue";
+import { APP_BASE_ROUTE } from "../router/utils";
+
+const props = defineProps({
+    selectedIndicator: {
+        type: String,
+        required: true
+    }
+});
+
+const router = useRouter();
+
+const resetView = () => {
+    let homePath = `/${APP_BASE_ROUTE}/${props.selectedIndicator}`;
+    let currentPath = router.currentRoute.value.fullPath;
+    homePath = homePath.replace(/\/+$/, "");
+    currentPath = currentPath.replace(/\/+$/, "");
```

Comment:
It should be reliable here to just check if selectedCountryId is set - would be a bit simpler than this.

Reply:
The problem is that when you're zoomed in but haven't selected any country, the path hasn't changed and nor has the selectedCountryId. However, using the updateBounds method to zoom out should fix all this.

Reply:
No, but if selectedCountryId is something other than empty, you should be able to reliably navigate to the base url. if it's already empty, you just need to updateBounds. if you just updated bounds when not empty, you wouldn't return to the global view - which is what you want. Maybe that's what you're saying though!